### DISCUSSION TITLE
Divide the archive into several parts

### DISCUSSION THREAD

**Discussion by daitkhealex** [spatie/laravel-backup#1372] [Ideas]
Good afternoon. Is it possible to add to the package the ability to divide a large zip archive into several parts for quick download. Some service providers reduce the download speed of the archive at large size. Thank you for your response.

**Comment by erikn69** [comment]
I might be wrong, but I don't think it's possible using php. At least I never encountered this possibility using [ZipArchive](https://www.php.net/ziparchive), if you have examples or can make a PR would be helpful

For now, you can do it from console [`split-zip-archive-into-multiple-blocks`](https://linuxconfig.org/how-to-split-zip-archive-into-multiple-blocks-of-a-specific-size)
